projects/ngneat/dialog/src/lib/dialog.component.ts

```diff
@@ -83,7 +82,7 @@ export class DialogComponent implements OnInit, OnDestroy {
     const backdrop = this.config.backdrop ? this.backdrop.nativeElement : this.document.body;
     const dialogElement = this.dialogElement.nativeElement;
 
-    const backdropClick$ = fromEvent<MouseEvent>(backdrop, 'mouseup').pipe(
+    const backdropClick$ = fromEvent<MouseEvent>(backdrop, 'click').pipe(
```

Comment:
@tonivj5 I wonder if there any reason you used `mouseup` here?

Reply:
Hmm, sorry I don't remember why I used mouseup instead of click, which sounds as the ideal one 😅